### ISSUE TITLE
NullPointerException

### ISSUE THREAD

**Issue by yer9ali** [cortinico/slidetoact#18]
```
Exception throw version 7
E/AndroidRuntime: FATAL EXCEPTION: main
                  Process: kz.orange.orangetaxi, PID: 25516
                  java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Class java.lang.Object.getClass()' on a null object reference
                      at android.graphics.drawable.AnimatedVectorDrawable$VectorDrawableAnimatorRT.createRTAnimator(AnimatedVectorDrawable.java:1262)
                      at android.graphics.drawable.AnimatedVectorDrawable$VectorDrawableAnimatorRT.parseAnimatorSet(AnimatedVectorDrawable.java:1220)
                      at android.graphics.drawable.AnimatedVectorDrawable$VectorDrawableAnimatorRT.init(AnimatedVectorDrawable.java:1187)
                      at android.graphics.drawable.AnimatedVectorDrawable.ensureAnimatorSet(AnimatedVectorDrawable.java:851)
                      at android.graphics.drawable.AnimatedVectorDrawable.start(AnimatedVectorDrawable.java:834)
                      at com.ncorti.slidetoact.SlideToActView.startTickAnimation(SlideToActView.kt:589)
                      at com.ncorti.slidetoact.SlideToActView.access$startTickAnimation(SlideToActView.kt:40)
                      at com.ncorti.slidetoact.SlideToActView$startAnimationComplete$5.onAnimationUpdate(SlideToActView.kt:451)
                      at android.animation.ValueAnimator.animateValue(ValueAnimator.java:1350)
                      at android.animation.ValueAnimator.setCurrentFraction(ValueAnimator.java:615)
                      at android.animation.ValueAnimator.setCurrentPlayTime(ValueAnimator.java:581)
                      at android.animation.ValueAnimator.start(ValueAnimator.java:1005)
                      at android.animation.ValueAnimator.start(ValueAnimator.java:1014)
                      at android.animation.AnimatorSet.start(AnimatorSet.java:678)
                      at android.animation.AnimatorSet.onChildAnimatorEnded(AnimatorSet.java:807)
                      at android.animation.AnimatorSet.-wrap1(AnimatorSet.java)
                      at android.animation.AnimatorSet$AnimatorSetListener.onAnimationEnd(AnimatorSet.java:784)
                      at android.animation.ValueAnimator.endAnimation(ValueAnimator.java:1153)
                      at android.animation.ValueAnimator.doAnimationFrame(ValueAnimator.java:1313)
                      at android.animation.AnimationHandler.doAnimationFrame(AnimationHandler.java:146)
                      at android.animation.AnimationHandler.-wrap2(AnimationHandler.java)
                      at android.animation.AnimationHandler$1.doFrame(AnimationHandler.java:54)
                      at android.view.Choreographer$CallbackRecord.run(Choreographer.java:872)
                      at android.view.Choreographer.doCallbacks(Choreographer.java:686)
                      at android.view.Choreographer.doFrame(Choreographer.java:618)
                      at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:860)
                      at android.os.Handler.handleCallback(Handler.java:751)
                      at android.os.Handler.dispatchMessage(Handler.java:95)
                      at android.os.Looper.loop(Looper.java:154)
                      at android.app.ActivityThread.main(ActivityThread.java:6119)
                      at java.lang.reflect.Method.invoke(Native Method)
                      at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
                      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
```

**Comment by cortinico** [comment]
Hey @yer9ali 
Thanks for the report:

Can you please add more details to this issue to help me debug it:

Library Version.
Android Version.
Emulator/Device specs.
Logs/Crash reports/Stacktraces.
Example Code/Link to repositories (if is open-source)

**Comment by yer9ali** [comment]
Library Version. Library Version. implementation 'com.ncorti:slidetoact:0.3.0'
Android Version. v 7.1.1
Emulator/Device specs. emulator Pixel 5,0'
Logs/Crash reports/Stacktraces:
E/AndroidRuntime: FATAL EXCEPTION: main
                  Process: kz.orange.orangetaxi, PID: 2923
                  java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Class java.lang.Object.getClass()' on a null object reference
                      at android.graphics.drawable.AnimatedVectorDrawable$VectorDrawableAnimatorRT.createRTAnimator(AnimatedVectorDrawable.java:1262)
                      at android.graphics.drawable.AnimatedVectorDrawable$VectorDrawableAnimatorRT.parseAnimatorSet(AnimatedVectorDrawable.java:1220)
                      at android.graphics.drawable.AnimatedVectorDrawable$VectorDrawableAnimatorRT.init(AnimatedVectorDrawable.java:1187)
                      at android.graphics.drawable.AnimatedVectorDrawable.ensureAnimatorSet(AnimatedVectorDrawable.java:851)
                      at android.graphics.drawable.AnimatedVectorDrawable.start(AnimatedVectorDrawable.java:834)
                      at com.ncorti.slidetoact.SlideToActView.startTickAnimation(SlideToActView.kt:589)
                      at com.ncorti.slidetoact.SlideToActView.access$startTickAnimation(SlideToActView.kt:40)
                      at com.ncorti.slidetoact.SlideToActView$startAnimationComplete$5.onAnimationUpdate(SlideToActView.kt:451)
                      at android.animation.ValueAnimator.animateValue(ValueAnimator.java:1350)
                      at android.animation.ValueAnimator.setCurrentFraction(ValueAnimator.java:615)
                      at android.animation.ValueAnimator.setCurrentPlayTime(ValueAnimator.java:581)
                      at android.animation.ValueAnimator.start(ValueAnimator.java:1005)
                      at android.animation.ValueAnimator.start(ValueAnimator.java:1014)
                      at android.animation.AnimatorSet.start(AnimatorSet.java:678)
                      at android.animation.AnimatorSet.onChildAnimatorEnded(AnimatorSet.java:807)
                      at android.animation.AnimatorSet.-wrap1(AnimatorSet.java)
                      at android.animation.AnimatorSet$AnimatorSetListener.onAnimationEnd(AnimatorSet.java:784)
                      at android.animation.ValueAnimator.endAnimation(ValueAnimator.java:1153)
                      at android.animation.ValueAnimator.doAnimationFrame(ValueAnimator.java:1313)
                      at android.animation.AnimationHandler.doAnimationFrame(AnimationHandler.java:146)
                      at android.animation.AnimationHandler.-wrap2(AnimationHandler.java)
                      at android.animation.AnimationHandler$1.doFrame(AnimationHandler.java:54)
                      at android.view.Choreographer$CallbackRecord.run(Choreographer.java:872)
                      at android.view.Choreographer.doCallbacks(Choreographer.java:686)
                      at android.view.Choreographer.doFrame(Choreographer.java:618)
                      at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:860)
                      at android.os.Handler.handleCallback(Handler.java:751)
                      at android.os.Handler.dispatchMessage(Handler.java:95)
                      at android.os.Looper.loop(Looper.java:154)
                      at android.app.ActivityThread.main(ActivityThread.java:6119)
                      at java.lang.reflect.Method.invoke(Native Method)
                      at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
                      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)

**Comment by cortinico** [comment]
Hey @yer9ali 
I tested on a Pixel 5' on Android 7.1.1 as from your report.

All works fine and I found no crashes (so I'm closing here).

I also googled a bit and it looks like your crash is related to `path` naming in the animated vector drawable: https://issuetracker.google.com/issues/37138817

Maybe you changed the `android:name` of the path in the `animated_ic_check.xml` file or in the `ic_check.xml` file? They should be the same otherwise the framework is not able to animate your path.
